tests/python/tests/job_workers/test_job_errors.py

```diff
@@ -76,8 +76,43 @@ def test_job_exception_error(self):
         ])
 
     def test_job_stack_overflow_error(self):
-        self.job_error_test_impl("stack_overflow", self.JOB_STACK_OVERFLOW_ERROR, data=[[1, 2, 3, 4, 5]], buffers=2)
-        self.kphp_server.assert_log([
+        error_code = self.JOB_STACK_OVERFLOW_ERROR
+        data = [[1, 2, 3, 4], [7, 9, 12]]
+        buffers = 4
+        stats_before = self.kphp_server.get_stats()
+        resp = self.kphp_server.http_post(
+        uri="/test_job_errors",
+        json={
+            "tag": "x2_with_error",
+            "error-type": "stack_overflow",
+            "data": data
+        })
+        self.assertEqual(resp.status_code, 200)
+
+        job_result = resp.json()["jobs-result"]
+        results = 2
+
+        eq = 0
+        got_error_code = ''
+        for i in range(results):
+            if job_result[i]["error_code"] == error_code:
+                eq += 1
+            else:
+                got_error_code = job_result[i]["error_code"]
+        if eq == 0:
+            self.assertEqual(got_error_code, error_code)
```

Comment:
Now, you overwrite `got_error_code` and compare here only the last one. It it intentional?

Reply:
Yes, because it's last wrong error code.